nix/k.nix

```diff
@@ -37,7 +39,9 @@ let
       "-Dllvm.backend.skip=true"
       "-Dhaskell.backend.skip=true"
     ];
-
+    
+    cmakeBuildType = if release then "Release" else "FastBuild";
+    
```

Comment:
This won't do anything here because the frontend is built with Maven instead of CMake. Also, the frontend is all in Java, so there is no LTO anyway.

```suggestion
```